src/test/java/org/apache/drillui/test/framework/pages/StoragePage.java

```diff
@@ -30,17 +30,31 @@
 
 public class StoragePage extends BasePage {
 
-  @FindBy(xpath = "//*[text()='Enabled Storage Plugins']/following-sibling::div[@class='table-responsive']")
+  // @FindBy(xpath = "//*[text()='Enabled Storage Plugins']/following-sibling::div[@class='table-responsive']")
+  @FindBy(xpath = "/html/body/div[2]/div[3]")
   private WebElement enabledStoragePlugins;
 
-  @FindBy(xpath = "//*[text()='Disabled Storage Plugins']/following-sibling::div[@class='table-responsive']")
+  // @FindBy(xpath = "//*[text()='Disabled Storage Plugins']/following-sibling::div[@class='table-responsive']")
+  @FindBy(xpath = "/html/body/div[2]/div[4]")
   private WebElement disabledStoragePlugins;
 
-  @FindBy(id = "storageName")
-  private WebElement newStoragePluginInput;
+  @FindBy(css = "button[data-target='#new-plugin-modal']")
+  private WebElement newStoragePluginDialog;
 
-  @FindBy(xpath = "//button[text()='Create']")
-  private WebElement newStoragePluginSubmit;
+  @FindBy(xpath = "//*[@id=\"configuration\"]")
+  private WebElement formTitle;
+
+  @FindBy(css = "input[placeholder='Storage Name']")
+  private WebElement newStoragePluginNameInput;
+
+  @FindBy(xpath = "//*[@id=\"editor\"]/textarea")
+  private WebElement newStoragePluginConfigInput;
+
+  @FindBy(xpath = "//*[@id=\"createForm\"]/div[2]/button[1]")
```

Comment:
```suggestion
  @FindBy(id = "createForm")
```
You can find the button further relatively to this element. Don't use indices in locators.

Reply:
Done